prow/prow_secrets.md

```diff
@@ -75,6 +81,5 @@ data:
 ```
 
 The `Secret` will be updated automatically when the secret value in gsm changed
-or the `ExternalSecret` is changed. The secret will not be deleted by kubernetes
-external secret, even in the case of `ExternalSecret` CR is deleted from the
-cluster. Deletion of `Secret` will need to be handled separately if desired.
+or the `ExternalSecret` is changed. when `ExternalSecret` CR is deleted from the
+cluster, the secret will be also be deleted by kubernetes external secret.
```

Comment:
Note that deleting the ExternalSecret from git source does not cause our deployment automation to delete the resource in cluster, so this still requires manual intervention.

Reply:
done